core/connectors.js

```diff
@@ -1020,6 +1020,13 @@ define(function() {
 			matches: ['*://*radiobonton.cz/*'],
 			js: ['connectors/v2/danceradiocz.js'],
 			version: 2
+		},
+
+		{
+			label: 'GrooveMP3',
+			matches: ['*://groovemp3.com/*'],
```

Comment:
_www_ subdomain is also valid, please include that: http://www.groovemp3.com/  

    matches: ['*://groovemp3.com/*', '*://www.groovemp3.com/*'],